int8 Nothing
---
float32 x
float32 y
float32 z
